library/network/src/modules/PortAliases.rb

```diff
@@ -1,137 +1,93 @@
-# ***************************************************************************
+# Copyright (c) [2013-2020] SUSE LLC
 #
-# Copyright (c) 2002 - 2012 Novell, Inc.
 # All Rights Reserved.
 #
-# This program is free software; you can redistribute it and/or
-# modify it under the terms of version 2 of the GNU General Public License as
-# published by the Free Software Foundation.
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
 #
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
-# GNU General Public License for more details.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
 #
-# You should have received a copy of the GNU General Public License
-# along with this program; if not, contact Novell, Inc.
-#
-# To contact Novell about this file by physical or electronic mail,
-# you may find current contact information at www.novell.com
-#
-# ***************************************************************************
-# File:  modules/PortAliases.ycp
-# Package:  Ports Aliases.
-# Summary:  Definition of Port Aliases.
-# Authors:  Lukas Ocilka <locilka@suse.cz>
-#
-# $Id$
-#
-# Global Definition of Port Aliases for services taken from /etc/services.
```

Comment:
It's weird because the documentation said that, but actually there are some services _pre-loaded_. I have preferred to be conservative and keep them as _KNOWN_SERVICES_, but seems that they are not needed at all.

:question:

Reply:
I can confirm that in my system all of them are also in the `/usr/etc/services` database.

Reply:
And finally I removed them. Please, have a look to https://github.com/yast/yast-yast2/pull/1026/commits/dd2118148c4fd03bb91721ccf25b9cc2fb825550